apcd-cms/src/apps/submissions/templates/view_submission_logs_modal.html

```diff
@@ -12,23 +12,21 @@ <h4 class="modal-title">View Logs</h4>
         <div class="modal-body">
           <div>
             <dl>
-              <h4>Logs</h4>
+              <h4 class="title">Logs</h4>
               {% for log in r.view_modal_content %}
-                <dd>          
+                <div class="modal-section">
                     <dl class="c-data-list--is-vert c-data-list--is-wide">
-                    <dt class="c-data-list__key">Log ID</dt>
-                    <dd class="c-data-list__value">{{log.log_id}}</dd>
-                    <dt class="c-data-list__key">Submitter ID</dt>
-                    <dd class="c-data-list__value">{{log.submitter_id}}</dd>                       
-                    <dt class="c-data-list__key">File Type</dt>                
-                    <dd class="c-data-list__value">{{log.file_type}}</dd>
-                    <dt class="c-data-list__key">Validation Suite</dt>                
-                    <dd class="c-data-list__value">{{log.validation_suite}}</dd>
-                    <dt class="c-data-list__key">Outcome</dt>                
-                    <dd class="c-data-list__value">{{log.outcome}}</dd>
+                        <dt class="c-data-list__key">Log ID</dt>
+                        <dd class="c-data-list__value">{{log.log_id}}</dd>                    
+                        <dt class="c-data-list__key">File Type</dt>                
```

Comment:
It looks like Submitter ID is no longer listed in the modal. Was that requested to be removed and this is intentional? Just double checking with the screen shot and my modal only has Log ID, file type, validation suite, and outcome listed per log.

Reply:
Sorry I attached an older screenshot. Yes Tracy asked me to remove that field